experiment/test/config.txt
2021-05-13-15:25:18

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:24

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:25

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:26

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:26

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:25:26

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:27:56

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:07

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:09

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu
pre_train: pixby/srtest/experiment/edsr_baseline_x4/model/model_best.pt
extend: .
n_resblocks: 16
n_feats: 64
res_scale: 1
shift_mean: True
dilation: False
precision: single
G0: 64
RDNkSize: 3
RDNconfig: B
n_resgroups: 10
reduction: 16
reset: False
test_every: 1000
epochs: 300
batch_size: 16
split_batch: 1
self_ensemble: False
test_only: True
gan_k: 1
lr: 0.0001
decay: 200
gamma: 0.5
optimizer: ADAM
momentum: 0.9
betas: (0.9, 0.999)
epsilon: 1e-08
weight_decay: 0
gclip: 0
loss: 1*L1
skip_threshold: 100000000.0
save: test
load: 
resume: 0
save_models: False
print_every: 100
save_results: True
save_gt: False

2021-05-13-15:28:10

debug: False
template: .
n_threads: 6
cpu: False
n_GPUs: 1
seed: 1
dir_data: /home/team1/EDSR-PyTorch/dataset/DIV2K
dir_demo: pixby/srtest/test
data_train: ['DIV2K']
data_test: ['Demo']
data_range: 1-800/801-810
ext: sep
scale: [4]
patch_size: 192
rgb_range: 255
n_colors: 3
chop: True
no_augment: False
model: EDSR
act: relu

backend/images/core/config.py
#! /usr/bin/env python
# coding=utf-8
from easydict import EasyDict as edict

from pathlib import Path
CURRENT_PATH = Path(__file__).resolve().parent.parent


__C                           = edict()
# Consumers can get config by: from config import cfg

cfg                           = __C

# YOLO options
__C.YOLO                      = edict()

__C.YOLO.CLASSES              = CURRENT_PATH / "data/classes/coco.names"
__C.YOLO.ANCHORS              = [12,16, 19,36, 40,28, 36,75, 76,55, 72,146, 142,110, 192,243, 459,401]
__C.YOLO.ANCHORS_V3           = [10,13, 16,30, 33,23, 30,61, 62,45, 59,119, 116,90, 156,198, 373,326]
__C.YOLO.ANCHORS_TINY         = [23,27, 37,58, 81,82, 81,82, 135,169, 344,319]
__C.YOLO.STRIDES              = [8, 16, 32]
__C.YOLO.STRIDES_TINY         = [16, 32]
__C.YOLO.XYSCALE              = [1.2, 1.1, 1.05]
__C.YOLO.XYSCALE_TINY         = [1.05, 1.05]
__C.YOLO.ANCHOR_PER_SCALE     = 3
__C.YOLO.IOU_LOSS_THRESH      = 0.5


# Train options
__C.TRAIN                     = edict()

__C.TRAIN.ANNOT_PATH          = "./data/dataset/val2017.txt"
__C.TRAIN.BATCH_SIZE          = 2
# __C.TRAIN.INPUT_SIZE            = [320, 352, 384, 416, 448, 480, 512, 544, 576, 608]
__C.TRAIN.INPUT_SIZE          = 416
__C.TRAIN.DATA_AUG            = True
__C.TRAIN.LR_INIT             = 1e-3
__C.TRAIN.LR_END              = 1e-6
__C.TRAIN.WARMUP_EPOCHS       = 2
__C.TRAIN.FISRT_STAGE_EPOCHS    = 20
__C.TRAIN.SECOND_STAGE_EPOCHS   = 30



# TEST options
__C.TEST                      = edict()

__C.TEST.ANNOT_PATH           = "./data/dataset/val2017.txt"
__C.TEST.BATCH_SIZE           = 2
__C.TEST.INPUT_SIZE           = 416
__C.TEST.DATA_AUG             = False
__C.TEST.DECTECTED_IMAGE_PATH = "./data/detection/"
__C.TEST.SCORE_THRESHOLD      = 0.25
__C.TEST.IOU_THRESHOLD        = 0.5


